core/src/channel_group/mod.rs

```diff
@@ -21,15 +24,29 @@ pub struct ChannelGroup {
     audio_params: AudioStreamParams,
 }
 
+/// Options for initializing a new ChannelGroup.
 pub struct ChannelGroupConfig {
+    /// Channel initialization options (same for all channels).
     pub channel_init_options: ChannelInitOptions,
+
+    /// Amount of VoiceChannel objects to be created.
+    /// (Number of MIDI channels)
     pub channel_count: u32,
+
+    /// A vector which specifies which of the created channels (indexes) will be used for drums.
+    /// For example in a conventional 16 MIDI channel setup where channel 10 is used for
+    /// drums, the vector would be set as \[9\] (counting from 0).
     pub drums_channels: Vec<u32>,
+
+    /// Parameters of the output audio.
     pub audio_params: AudioStreamParams,
+
+    /// Whether or not to use a threadpool to render voices.
```

Comment:
Maybe it's good to clarify that all channels are handled in their own threads still, this setting adds more fine-grained threading per key rather than per channel